src/common/scaled-rect-buffer.c

```diff
@@ -58,6 +58,8 @@ _create_buffer(struct scaled_scene_buffer *scaled_buffer, double scale)
 	set_cairo_color(cairo, self->border_color);
 	cairo_stroke(cairo);
 
+	cairo_destroy(cairo);
```

Comment:
Do we need a cairo_surface_flush() here? (Not sure, I just noticed we have one in most of the other cases.)

Reply:
Nice catch. I'll add it.